cmd/fleetctl/config.go

```diff
@@ -235,6 +267,13 @@ func configSetCommand() *cli.Command {
 				Destination: &flURLPrefix,
 				Usage:       "Specify URL Prefix to use with Fleet server (copy from server configuration)",
 			},
+			&cli.StringSliceFlag{
+				Name:        "custom-header",
+				EnvVars:     []string{"CUSTOM_HEADER"},
```

Comment:
Yeah good call. I don't think it's a big issue because once set it's saved in the fleetctl config, so I think it's unlikely to ever be used via env vars but I'll document it.